Source/monstdat.h

```diff
@@ -5,7 +5,9 @@
  */
 #pragma once
 
+#include <cstddef>
 #include <cstdint>
+#include <string>

```

Comment:
What are these for?

Reply:
There are a couple of structs at the bottom of this file that use these. They previously worked due to transitive includes from effects.h

Reply:
I see it now, via `textdat.h`.